### PR TITLE
Documentation with docsify

### PR DESCRIPTION
**Pull request by tpluscode**
Based off @vhf `documentation` branch I created a docs dir which will run as GitHub pages.

The documentation pages are build-less, all markdown etc running in the browser only. Viewable locally with `npm run docs:serve`.

@vhf, I flexed the TBBT example and moved it to a separate page as it already seems a little verbose for the index page. I replaced it with something smaller on the front page

@vhf @l00mi I also added some ideas for sub-pages in the side menu. Feel free to change or add more

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+```
+npm i -S clownface
+```
+
+To start using it, an instance of [RDF/JS `DatasetCore`](https://rdf.js.org/dataset-spec/#datasetcore-interface) must be
+provided to the exported factory method
+
+<run-kit>
+
+```js
```
Comment (l00mi) [inline docs/README.md]: I love the idea of creating a graph for the first example. Much cleaner and independent of loaders and stuff!
Further I propose to remove the name space part `{ foaf }` as it mixes concepts and might confuse. Its good to have it in the deepdive though!

**Comment by tpluscode** [comment]
Docsify adds a GitHub Octocat link in the corner

File: docs/README.md (hunk 1 above)
Comment (tpluscode) [reply docs/README.md]: Fair enough

File: package.json [hunk 2]
```diff
@@ -40,5 +42,10 @@
     "exclude": [
       "test/*.js"
     ]
+  },
+  "standard": {
+    "ignore": [
+      "docs/"
+    ]
```
Comment (bergos) [inline package.json]: I guess this was only needed for the generated API stuff and is not need anymore. Can we remove it?

**Pull request merged**